model-engine/model_engine_server/api/llms_v1.py

```diff
@@ -77,7 +78,21 @@
 from model_engine_server.domain.use_cases.model_bundle_use_cases import CreateModelBundleV2UseCase
 from sse_starlette.sse import EventSourceResponse
 
-llm_router_v1 = APIRouter(prefix="/v1/llm")
+
+async def record_route_call(
```

Comment:
We could consider adding this at the highest level so all model-engine routes get it (e.g bundles/endpoits).